CMakeLists.txt

```diff
@@ -41,13 +41,23 @@ add_library(vkgc_headers INTERFACE)
 
 ### Options that affect the headers ####################################################################################
 option(LLPC_ENABLE_SHADER_CACHE "Enable experimental shader cache" OFF)
+
 if(LLPC_ENABLE_SHADER_CACHE)
-    target_compile_definitions(vkgc_headers INTERFACE LLPC_ENABLE_SHADER_CACHE=1)
+    target_compile_definitions(vkgc_headers INTERFACE LLPC_ENABLE_SHADER_CACHE)
 endif()
+
 if (LLPC_CLIENT_INTERFACE_MAJOR_VERSION)
     target_compile_definitions(vkgc_headers INTERFACE LLPC_CLIENT_INTERFACE_MAJOR_VERSION=${LLPC_CLIENT_INTERFACE_MAJOR_VERSION})
+else()
+    # LLPC is not compiled, so fall back to the latest version
+    target_compile_definitions(vkgc_headers INTERFACE LLPC_CLIENT_INTERFACE_MAJOR_VERSION=LLPC_INTERFACE_MAJOR_VERSION)
 endif()
 
+#if VKI_KHR_FRAGMENT_SHADER_BARYCENTRIC
```

Comment:
Do we need these comments?

Reply:
Sorry, these code should not be here, I'll update and remove them.